src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/data/AdjudicationPunishmentMappingDto.kt

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.data
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.Size
+import uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.AdjudicationPunishmentMapping
+import java.time.LocalDateTime
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Adjudication punishment (aka award) mapping")
+data class AdjudicationPunishmentMappingDto(
+
+  @Schema(description = "NOMIS booking id", required = true, example = "123456")
+  val nomisBookingId: Long,
+
+  @Schema(description = "NOMIS sanction sequence", required = true, example = "4")
+  val nomisSanctionSequence: Int,
+
+  @Schema(description = "DPS hearing id", required = true, example = "123456")
```

Comment:
Not hearing id - punishment id?

Reply:
yes, I'll submit a new PR after this is merged